### ISSUE TITLE
Update Tested specs in readme

### ISSUE THREAD

**Issue by ponceta** [qwat/qwat-data-model#259]
Ref: https://github.com/qwat/qwat-data-model/pull/248

It should be adequate to assess at least tested versions of tools at least for the latest release, and mention in changelog (release tab) when submodules or external libraires are updated.

Actually tested is 
 - postgres 9.6
 - Python 3.5.5
 - pum 0.6.7
 - pip 9.0.1

**Comment by haubourg** [comment]
Agreed. And QGIS too is concerned because we use some features >= QGIS 2.16

let's reference that in #250

**Comment by lbartoletti** [comment]
That sounds good to me now. Can we close the ticket?
If necessary, one can be opened on qwat/docs.
